server/scripts/remove-duplicate-locations.ts

```diff
@@ -161,22 +167,36 @@ async function doChanges(plans, persist = false) {
   return { removed, updated };
 }
 
-async function main(args) {
+export async function main({
+  unpad = false,
+  commit = false,
+  system = [],
+  ..._
+}: {
+  unpad?: boolean;
+  commit?: boolean;
+  system?: (string | number)[];
+}): Promise<void> {
+  assert.ok(
+    system.every((s) => typeof s === "string"),
+    "Systems must all be strings"
+  );
```

Comment:
This has never been an issue in practice, but the TS compiler reminded me that Yargs arrays are always `string|number`; you can’t really stop it from parsing the values of an array option as numbers, AFAICT.